tests/smoketest/mixins.py

```diff
@@ -83,6 +83,35 @@ def test_search(self, r):
         assert "doc2" in docs
         assert "doc1" in docs
 
+    def test_versions_match(self, r):
+        stack_dockloader(r)
+        modmap = {
+            "redisearch": "search",
+            "redisgraph": "graph",
+            "redistimeseries": "timeseries",
+            "redisbloom": "bf",
+            "rejson": "rejson",
+        }
+
+        with open(CONFIGYAML, "r") as fp:
+            data = yaml.load(fp, yaml.SafeLoader)
+            versions = data.get("versions")
+
+        modlist = r.module_list()
+        modules = {m.get("name").lower(): m.get("ver") for m in modlist}
+        for k, v in modmap.items():
+            yamlversion = versions.get(k)
+
+            # remap
+            parts = yamlversion.split(".")
+            last = parts[2]
+            if len(parts[2]) == 1:
+                last = f"{parts[2]}0"
+            version = f"{parts[0]}0{parts[1]}{last}"
```

Comment:
Major version is always a single digit?

Reply:
So far... when decoding what we have, yes.